backend/Controllers/RecipeController.js

```diff
@@ -219,39 +271,40 @@ const deleteRecipe = async (req, res) => {
           'X-GitHub-Api-Version': '2022-11-28'
         }
         });
-        const sha=response.data.sha;
         // Getting the SHA key so that it can assist in deletion
+        const sha=response.data.sha;
         await octokit.request('DELETE /repos/{owner}/{repo}/contents/{path}', {
         owner: owner,
         repo: repo,
         path: result,
         message: 'deleted the image',
```

Comment:
@sadath2001 one comment,
Can you change this commit message to `deleted the image - {image-name}`
It will be helpful to get insights easily from history of github commit.